builtin/providers/datadog/resource_datadog_user.go

```diff
@@ -107,7 +108,7 @@ func resourceDatadogUserRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	log.Printf("[DEBUG] user: %v", u)
+	log.Printf("[DEBUG] user: %s", spew.Sdump(u))
```

Comment:
I only left it in as the `#v` output is useless - it's just a bunch of memory addresses. Should we remove the entire line?